Assets/Mirror/Tests/Editor/NetworkServerTest.cs

```diff
@@ -349,6 +349,44 @@ public void RemoveConnectionTest()
             Assert.That(NetworkServer.connections.Count, Is.EqualTo(0));
         }
 
+        [Test]
+        public void DisconnectTimeoutTest()
```

Comment:
This should be a playmode test so that you can use `yield return new WaitFroSeconds(10);`

Reply:
It's a playmode test now.